src/azure-cli/azure/cli/command_modules/containerapp/custom.py

```diff
@@ -392,7 +392,7 @@ def update_containerapp_logic(cmd,
                         e["value"] = ""
 
     update_map = {}
-    update_map['scale'] = min_replicas or max_replicas or scale_rule_name
+    update_map['scale'] = min_replicas is not None or max_replicas is not None or scale_rule_name is not None
```

Comment:
what about scalue_rule_name with empty str

Reply:
Revert the change about `scale_rule_name` because it is not related to the fix.